public/index.html

```diff
@@ -25,6 +25,12 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>TDEX Dashboard</title>
+    <script>
+      if (!'__TAURI__' in window) {
+        window.USE_PROXY = __USE_PROXY__;
+        window.PROXY_URL = __PROXY_URL__;
+      }
```

Comment:
Why the window vars are set only if not in a tauri environment?

Maybe this should become:
```suggestion
      if (process.env.USE_PROXY) {
        window.USE_PROXY = process.env.USE_PROXY;
      }
      if (process.env.PROXY_URL) {
        window.PROXY_URL = process.env.PROXY_URL;
      }
```
?

Reply:
browser SPAs does not have process.env at all. So we do  a trick to inject env vars into window loading the HTML page before serving 

https://github.com/tdex-network/tdex-dashboard/blob/docker/Dockerfile.web#L36-L38

Reply:
the check for TAURI is that in that case we ALWAYS use the proxy, so is pointless to expect those vars